scripts/deploy-kubevirt-and-cdi.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+function latest_version() {
+  local repo="$1"
+  curl -s https://api.github.com/repos/kubevirt/${repo}/releases/latest | \
+    grep '"tag_name":' | \
+    sed -E 's/.*"([^"]+)".*/\1/'
+}
+
+
+# Deploying kubevirt
+KUBEVIRT_VERSION=$(latest_version "kubevirt")
```

Comment:
Can you just use `https://storage.googleapis.com/kubevirt-prow/devel/release/kubevirt/kubevirt/stable.txt` as we do elsewhere?

https://github.com/kubevirt/common-instancetypes/blob/d994341f28036818efad92e918a5ca9a359986a1/scripts/kubevirtci.sh#L53

Reply:
Done.